app/controllers/provider_foreman_controller.rb

```diff
@@ -249,8 +249,14 @@ def show(id = nil)
     @display = params[:display] || "main"
     @lastaction = "show"
     @showtype = "config"
-    @record =
-      find_record(configuration_profile_record? ? ConfigurationProfile : ConfiguredSystem, id || params[:id])
+    if configuration_profile_record?
+      @record = find_record(ConfigurationProfile, id || params[:id])
```

Comment:
Are you intentionally only setting `@record` for `ConfigurationProfile`s?  It used to set `@record` for `ConfigurationProfile` or `ConfiguredSystem`.